src/game-logic/reducers/waterPlot.js

```diff
@@ -0,0 +1,23 @@
+import { getPlotContentType } from '../../utils'
+import { itemType } from '../../enums'
+
+import { modifyFieldPlotAt } from './modifyFieldPlotAt'
+
+/**
+ * @param {farmhand.state} state
+ * @param {number} x
+ * @param {number} y
+ * @returns {farmhand.state}
+ */
+export const waterPlot = (state, x, y) => {
+  const plotContent = state.field[y][x]
+
+  if (!plotContent || getPlotContentType(plotContent) !== itemType.CROP) {
```

Comment:
positive checks read a bit more easily to me here

```suggestion
const canBeWatered = plotContent && getPlotContentType(plotContent) === itemType.CROP

  if (canBeWatered === false) {
```